src/discover/getDiscover.ts

```diff
@@ -30,7 +30,7 @@ export async function getMovieInfo(): Promise<AxiosResponse<DiscoverDetails>> {
   }
 }
 
-export async function getTvInfo(): Promise<AxiosResponse<DiscoverDetails>> {
+export async function getTvShowlist(): Promise<AxiosResponse<MovieDetails>> {
```

Comment:
return  type: Promise<AxiosResponse<TvShows>> may be?

Reply:
nope it return type is `Promise<AxiosResponse<MovieDetails>>`

Reply:
Is your function supposed to get MovieDetails or TvShowsList?